src/ng/q.js

```diff
@@ -217,20 +217,51 @@
  * @returns {Promise} The newly created promise.
  */
 function $QProvider() {
-
+  var errorOnUnhandledRejections = true;
   this.$get = ['$rootScope', '$exceptionHandler', function($rootScope, $exceptionHandler) {
     return qFactory(function(callback) {
       $rootScope.$evalAsync(callback);
-    }, $exceptionHandler);
+    }, $exceptionHandler, errorOnUnhandledRejections);
   }];
+
+  /**
+   * @ngdoc method
+   * @name $qProvider#errorOnUnhandledRejections
```

Comment:
I wonder where this method shows up in the docs (considering there are no docs for `$qProvider`).
I think there should "standalone" docs for `$qProvider` (even if just containing this method), but it can be tackled in a follow-up PR.